### DISCUSSION TITLE
Mehrmalige Passworteingabe erforderlich

### DISCUSSION THREAD

**Discussion by dkummer63** [SafeExamBrowser/seb-win-refactoring#411] [Q&A]
Ich bin in Vorbereitung der Abschlussprüfung an unserer Schule (190 Schüler) und habe den Schülern die SEB-Datei zukommen lassen, die entsprechend konfiguriert ist. Auf Mac konnte ich alle SEB-Dateien problemlos testen, aber auf meinem Windowsgerät (Win.11) muss ich das Passwort drei oder mehrmals eingeben, bis es akzeptiert wird. Einige Male hat es erst funktioniert, nachdem ich das Passwort eingab, mit dem die SEB-Datei erstmalig verwendet wurde und dann erst wurde das aktuelle Passwort akzeptiert. Was mache ich falsch?
Danke für jede Hilfe

**Comment by dbuechel** [comment]
Die Passwortabfrage sollte problemlos funktionieren, wir hatten diesbezüglich auch noch nie Supportanfragen. Stellen Sie bitte sicher, dass das Eingabefeld fokussiert ist und dass das korrekte Tastaturlayout sowie die Umschalttaste etc. aktiviert resp. nicht aktiviert sind.

Bei SEB-Konfigurationen gibt es zudem verschiedene Anwendungsfälle, einer davon ist die lokale SEB-Konfiguration (versus Prüfungskonfiguration), siehe https://safeexambrowser.org/windows/win_usermanual_en.html#ConfigFilePane. Da könnte es ggf. sein, dass statt der Prüfungskonfiguration die lokale Konfiguration (oder vice versa) gestartet wurde, welche u.U. ein anderes Passwort hat.

Sollte der Fehler weiterhin bestehen, wären wir froh, wenn Sie ein Video oder Fotos des genauen Ablaufs machen könnten, damit wir die Sache besser nachvollziehen können.

**Comment by dbuechel** [comment]
@dkummer63 Ich habe Ihren Kommentar gelöscht, da leider auch Antworten per E-Mail direkt auf GitHub gepostet werden. Das Video konnte ich herunterladen und werde es mir noch ansehen, eine Konfiguration konnte ich aber in Ihrer Antwort nicht finden.

**Reply by dbuechel** [reply]
@dkummer63 Vielen Dank, die Datei habe ich nun herunterladen können. Ja genau, auch wenn Sie per E-Mail antworten wird das gesamte Mail automatisch auf GitHub gepostet.

**Comment by dbuechel** [comment]
Vielen Dank für das Video, nun ist mir klar, was das Problem zu sein scheint: Auf Ihrem System hat es eine lokale SEB-Konfiguration, welche mit einem Passwort geschützt ist. Die Konfiguration der Studentin ist eine Konfiguration für das Setzen der lokalen Konfiguration (siehe https://safeexambrowser.org/windows/win_usermanual_en.html#ConfigFilePane) und ist darüber hinaus ebenfalls mit einem Passwort geschützt.

Dies ist ein besonderer Anwendungsfall, welcher SEB wie in der Benutzeranleitung angedeutet abhandelt:

1. Abfragen des Passwortes für die **lokale Konfiguration** (siehe Video bei 00:45: "Bitte geben Sie das Passwort für die **lokale Konfiguration** ein").
2. Es wird die lokale Konfiguration geladen.
3. Falls das Passwort für die zu startende, neue lokale Konfiguration nicht gleich ist wie bei der bereits bestehenden lokalen Konfiguration, dann wird nun dieses Passwort auch noch abgefragt (siehe Video bei 01:26: "Bitte geben Sie das Passwort für die **ausgewählte Konfiguration** ein").
4. Die bestehende lokale Konfiguration wird mit der neuen lokalen Konfiguration überschrieben (es sei denn, die bestehende lokale Konfiguration ist im Program Data Verzeichnis).

**Comment by dkummer63** [comment]
Das ist hilfreich. Ich habe nicht beachtet, dass da jeweils etwas unterschiedliches steht. Das beruhigt mich für die Maturprüfung morgen. Besten Dank und noch einen guten Tag
Daniel Kummer
